WhosTalking/Windows/ConfigWindow.cs

```diff
@@ -293,18 +293,32 @@ public sealed class ConfigWindow: Window, IDisposable {
                 ImGui.SetNextItemWidth(200);
                 ImGui.Combo(
                     "###IdsInCall",
-                    ref this.idInCall,
+                    ref this.idInCallIdx,
                     _idsWithName,
                     _idsWithName.Length
                 );
 
-                // Button to add entry to list
+                // Add entry to list
                 ImGui.TableNextColumn();
                 if (ImGui.Button("Add Entry")) {
+                    var _charaName = (this.playerInPartyIdx < playersInParty.Count) ? playersInParty[this.playerInPartyIdx] : "";
+                    var _discId = (this.playerInPartyIdx < idsInCall.Count) ? idsInCall[this.idInCallIdx] : "";
+
+                    var i = this.individualAssignments.Count;
+                    this.individualAssignments.Add(new AssignmentEntry());
+
+                    this.individualAssignments[i].CharacterName = _charaName;
+                    this.individualAssignments[i].DiscordId = _discId;
+                }
+
+                // Blank entry button
+                ImGui.SameLine();
+                if (ImGui.Button("Add blank Entry")) {
                     var i = this.individualAssignments.Count;
                     this.individualAssignments.Add(new AssignmentEntry());
-                    this.individualAssignments[i].CharacterName = playersInParty[this.playerInParty];
-                    this.individualAssignments[i].DiscordId = idsInCall[this.idInCall];
+
+                    this.individualAssignments[i].CharacterName = "";
+                    this.individualAssignments[i].DiscordId = "";
                 }
```

Comment:
this is what the "add manual assignment" button just below does, right?

Reply:
ngl i forgot about that button
up to you which to keep

Reply:
yeag true

i'm just gonna merge this, the ui is a mess all around and it can get cleaned up later